src/com/javafee/task/bankaccount/main.java

```diff
@@ -0,0 +1,35 @@
+package src.com.javafee.task.bankaccount;
+
+public class main {
+
```

Comment:
Please format the code and remove all unnecessary white spaces/characters etc. Keep your code short, simple, and clean. :)

Remember about the conventions i.e. class names begin with an upper letter, use the camel case instead of the snake case.